JS/Lesson31/saffet/my-app/src/components/CreateTask/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(task);
+
+    onCreateProps(task);
+    setTask("");
+  }
+
+  return (
+    <>
+      <h2>Create a task in below</h2>
+      <form className="form" action="">
+        <input onChange={handleInputChange} className="form-input" type="text" />
+        <button onClick={handleSubmit} className="create-btn">Save</button>
```

Comment:
If you add onSubmit event handeler to the form element itself, you won't need to any event handler to this button. Just make sure to set the type of this button to "submit".
```suggestion
        <button type="submit" className="create-btn">Save</button>
```